docs/genai/development/working-with-rag.ipynb

```diff
@@ -66,30 +66,9 @@
    "source": [
     "## Install a vector DB and dependencies\n",
     "\n",
-    "As mentioned above, this demo uses [Milvus](https://milvus.io) -&mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors. It can be installed via [docker-compose](https://milvus.io/docs/v2.0.x/install_standalone-docker.md) or [Kubernetes](https://milvus.io/docs/v2.0.x/install_standalone-helm.md) depending on your MLRun installation method.\n",
+    "As mentioned above, this demo uses [Milvus](https://milvus.io) -&mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors. It can be installed via [docker-compose](https://milvus.io/docs/v2.0.x/install_standalone-docker.md) or [Kubernetes](https://milvus.io/docs/v2.0.x/install_standalone-helm.md) depending on your MLRun installation method. In this demo, Milvus has been installed in our K8s cluster and the service is available at `http://milvus.default.svc.cluster.local:19530`.\n",
     "\n",
-    "This is a simple excerpt to install Milvus standalone via [Helm](https://helm.sh) without persistence &mdash; this is **NOT** for production usage:\n",
-    "```\n",
-    "helm repo add milvus https://milvus-io.github.io/milvus-helm/\n",
-    "helm repo update\n",
-    "helm install milvus milvus/milvus \\\n",
-    "    --set cluster.enabled=false \\\n",
-    "    --set etcd.replicaCount=1 \\\n",
-    "    --set minio.mode=standalone \\\n",
-    "    --set minio.persistence.enabled=false \\\n",
-    "    --set etcd.persistence.enabled=false \\\n",
-    "    --set pulsar.enabled=false \\\n",
-    "    --set standalone.persistence.enabled=false\n",
-    "```\n",
-    "\n",
-    "Additionally, if you are using Ollama, it can be installed on your [local machine](https://ollama.com/download) or [Kubernetes](https://github.com/otwld/ollama-helm) depending on your MLRun installation method.\n",
-    "\n",
-    "This is a simple example to install Ollama via [Helm](https://helm.sh):\n",
-    "```\n",
-    "helm repo add ollama-helm https://otwld.github.io/ollama-helm/\n",
-    "helm repo update\n",
-    "helm install ollama ollama-helm/ollama --set ollama.models={llama3}\n",
-    "```"
+    "Additionally, if you are using Ollama, it can be installed on your [local machine](https://ollama.com/download) or [Kubernetes](https://github.com/otwld/ollama-helm) depending on your MLRun installation method. Ollama has been installed and the service is available at `http://ollama.default.svc.cluster.local:11434`"
```

Comment:
"Additionally, if you are using Ollama, it can be installed on your [local machine](https://ollama.com/download) or [Kubernetes](https://github.com/otwld/ollama-helm) depending on your MLRun installation method. In this example, Ollama has been installed and the service is available at `http://ollama.default.svc.cluster.local:11434`"